examples/with-nextjs-and-clerk-auth/src/lib/monite-api/demo-data-generator/generate-entity.ts

```diff
@@ -99,7 +100,7 @@ export const generateEntity = async (
 
     const receivablesService = new ReceivablesService(serviceConstructorProps);
 
-    /** Create 15'ing Receivables with a type `invoice` */
+    /** Create 20'ing Receivables with a type `invoice` */
```

Comment:
[not required] Maybe I've over tsdoc it :) I think this TSDoc may be removed

Reply:
☑︎